### ISSUE TITLE
Make sure all Zoom, SlidesLive, and other links are correct in schedule

### ISSUE THREAD

**Issue by corcra** [acm-chil/acm-chil-website#45]
Sanity-checking that the website behaves as expected, this task just requires someone with time to go through and make sure there aren't bugs in the links.

- [ ] Zoom links (x-ref spreadsheet)

- [ ] SlidesLive (just check if it goes where expected)

- [ ] Link to item-specific page (just check if it goes where expected)
- [ ] (when we have them) RocketChat links to specific channels (check if it goes where expected)

**Comment by tnaumann** [comment]
@tasmie, can EE do these link checks?

**Comment by tasmie** [comment]
@corcra I went through the Pre-recorded page:

**Keynotes**
@tnaumann did we want to add abstracts? 
Everyone's links work except for Sherri's. I'm taken to this [page](https://www.chilconference.org/speaker_3.html) when I click on the title but when I click on "SlidesLive" it says presentation can't be found. This is probably something Monika can address. 

**Tutorials**
All links work

**Research Talks**
All links work 

**Workshop Spotlights**
A number of presentations require sign in for SlidesLive when you click on the title and/or SlidesLive link; Monika should be able to help. 

WS20 (https://www.chilconference.org/workshop_ws20.html) was actually withdrawn; we can remove this one. 
WS24 - still has the password on it- need to email Monika 
WS25 (https://www.chilconference.org/workshop_ws25.html) the title is different from what's on the presentation

**Doctoral Symposium**
All links work

**Comment by tasmie** [comment]
There are columns for Q&A for each of the talk types - is this live (Zoom) and chat Q&A (rocketchat)?

**Comment by corcra** [comment]
Thanks for going through this @tasmie !

- Re: Sherri Rose, that was a typo in the SlidesLive link, so thanks for catching it! I've fixed it now.
- I've removed WS20
- For WS25 I took the title from what's in the workshop data CSV, I don't know which is more "authoritative".

Regarding the columns for Q&A, do you mean the "Q&A Session" column? That column contains the time of the corresponding event in the schedule for that item - I was assuming these are all live Zoom Q&A sessions.

**Comment by tnaumann** [comment]
@tasmie if WS24 is the one where they wanted the password, don't have Monika remove unless you confirm with Shalmali that the authors are okay with that first.

**Comment by tasmie** [comment]
Thanks @corcra! I asked Monika about the workshop talks but I just wanted to make sure that that you had the correct embed codes: https://docs.google.com/spreadsheets/d/1gmGOE1sAHEno_tcWwFrmlQph6DbK59ofcngtwInE8J8/edit#gid=0

**Comment by tasmie** [comment]
Here's the detailed list of discrepancies on the pre-recorded talks page: 
Tutorials: 

“Public Health Datasets for Deep Learning: Challenges and Opportunities”

§  Opens “Nightingale: Building the future of medicine”

§  Authors are correct, but tutorial titles do not match


Workshop Spotlights:

The below sessions all take us to a login screen for SlidesLive

Learning Representations for Prediction of Next Patient State
Differentially Private Survival Function Estimation
A Multi-Task Learning Approach to Personalized Progression Modeling
Improved Patient Classification with Hierarchical Language Model Pretraining over Clinical Notes
Health change detection using temporal transductive learning
Cost-Sensitive Feature Selection Using Bayesian Optimization           
A large-scale Twitter dataset for drug safety applications mined from publicly existing resources
ClinicalBERT: Modeling Clinical Notes and Predicting Hospital Readmission
Mining Dynamic Problem Lists from Clinical Notes for the Interpretable Prediction of Adverse Outcomes
Deep Transfer Learning for Physiological Signals
Open Set Medical Diagnosis
Generation of Differentially Private Heterogeneous Synthetic Electronic Health Records using GANs
Integrating Physiological Time Series and Clinical Notes with Deep Learning for Improved ICU Mortality 
 

Automated Medical Coding using BERT: Benchmarking Deep Learning in the Face of Subjective Labels
Just asks for password, not an email address like the other workshops mentioned above
A Comprehensive Evaluation of Multitask Representation Learning on EHR Data
Titles do not match

**Comment by tasmie** [comment]
- [ ] Change workshop 25 title to: Distracted Multi-task Learning: Addressing Negative Transfer with Fine-tuning on EHR Time-series Data

**Comment by corcra** [comment]
Pages should have embedded rocketchat now, so we need to check that they go to the expected channel.

**Comment by tnaumann** [comment]
@corcra This is really cool.
@tasmie Can EE test this, both with respect to what happens when you're signed in and when you're signed out.

@tasmie Are the discrepancies in the comment above addressed? If not, can we get a concise list of what's remaining to be done so that we can start closing these out?

**Comment by corcra** [comment]
My understanding from the meeting earlier is that the SlidesLive situation is all resolved (wrt things which were previously asking one to log in).

I've updated the name of WS25.

I don't know what we should do about the title discrepancy for the public health dataset tutorial.

**Comment by tasmie** [comment]
SlidesLive situation is resolved. No need to re-order the public health dataset, those are going to be re-ordered on the minute by minute + speaker spreadsheet. Something we should a little more clear are the zoom rooms. Instead of using "Zoom" or "Zoom (as above), could we use Zoom room number, i.e. "Zoom 1" ? They are listed in the minute by minute here: https://docs.google.com/spreadsheets/d/1BRCCVjyUiWGI2s8os8-F5gY4mB9U6f70j9XCs_yiX2w/edit#gid=452609008
